db/tests/types/test_email.py

```diff
@@ -3,52 +3,36 @@
 from sqlalchemy import text, select, func, Table, Column, MetaData
 from sqlalchemy.exc import IntegrityError
 from db.engine import _add_custom_types_to_engine
-from db.types import install
+from db.tests.types import fixtures
 from db.types import email
 
 
-@pytest.fixture
-def engine_type_schema(engine):
-    install.create_type_schema(engine)
-    yield engine
-    with engine.begin() as conn:
-        conn.execute(
-            text(f"DROP SCHEMA IF EXISTS {install.base.SCHEMA} CASCADE;")
-        )
-
-
-@pytest.fixture
-def engine_email_type(engine_type_schema):
-    email.create_email_type(engine_type_schema)
-    return engine_type_schema
-
-
-@pytest.fixture
-def engine_with_app(engine_email_type):
-    app_schema = "test_app"
-    with engine_email_type.begin() as conn:
-        conn.execute(text(f"CREATE SCHEMA {app_schema};"))
-    yield engine_email_type, app_schema
-    with engine_email_type.begin() as conn:
-        conn.execute(text(f"DROP SCHEMA {app_schema} CASCADE;"))
+# We need to set these variables when the file loads, or pytest can't
+# properly detect the fixtures.  Importing them directly results in a
+# flake8 unused import error.
```

Comment:
Can't you just do a `#noqa` comment?

Reply:
Unfortunately, the number of #noqa comments necessary is large, since there's a sort of cascade of flake8 problems (i.e., importing them directly, then ignoring that line causes a giant pile of F811 -- redefinition of unused function errors wherever the fixtures are used.  It seemed cleaner to just use this pattern.  I've updated the comment to clarify that a bit.

Reply:
Thanks, sounds good.